modules/nf-core/crabz/compress/main.nf

```diff
@@ -0,0 +1,50 @@
+process CRABZ_COMPRESS {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/crabz:0.9.0':
+        'biocontainers/crabz:0.9.0' }"
+
+    input:
+    tuple val(meta), path(file)
+
+    output:
+    tuple val(meta), path("*.gz"), emit: archive
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${file}"
+    """
+    crabz \\
+        $args \\
+        -Q \\
+        -p $task.cpus \\
+        -o ${prefix}.gz \\
+        $file
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        crabz: \$(crabz --version |& sed 's/[^:]*://')
+    END_VERSIONS
+    """
+
+    stub:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${file}"
+    """
+    touch ${prefix}.gz
```

Comment:
For the stub test you will need to replace this because there is currently a bug with gzipped files 
```suggestion
    echo "" | gzip > ${prefix}.gz
```